vms/vms/urls.py

```diff
@@ -23,3 +26,6 @@
     url(r'^volunteer/', include('volunteer.urls', namespace='volunteer')),
 )
 
+if settings.DEBUG:
+    urlpatterns += staticfiles_urlpatterns()
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
These changes are not relevant to this issue. Kindly remove them.
When you start working on a new issue, checkout branch from develop (update develop if required) then start working on the issue.